mkt/api/tests/__init__.py

```diff
@@ -19,7 +21,8 @@ def _allowed_verbs(self, url, allowed):
             if verb in allowed:
                 continue
             try:
-                res = getattr(self.client, verb)(url)
+                with transaction.atomic():
```

Comment:
Why ? It's inside a `TestCase` class (not a `TransactionTestCase`), I'm not sure this will behave correctly...